web/js/map-OpenLayers.js

```diff
@@ -90,6 +91,34 @@ function fms_markers_list(pins, transform) {
     return markers;
 }
 
+function fms_marker_size_for_zoom(zoom) {
+    // zoomOffset seems to change between pages, so check that the zoom level
+    // we're working with is in the expected range
+    if (zoom < fixmystreet.zoomOffset) {
+        zoom += fixmystreet.zoomOffset;
+    }
+    var cobrand = $('meta[name="cobrand"]').attr('content');
+    if (cobrand === 'oxfordshire') {
```

Comment:
I would be happy for this to apply to all cobrands, as it makes it better for all, unless there's a reason not to do so.